src/sections/ping.tsx

```diff
@@ -0,0 +1,113 @@
+import { useEffect, useState } from 'react';
+import { nodeActionsAsync } from '../store/slices/node';
+import { useAppDispatch, useAppSelector } from '../store';
+
+import AbbreviatedPeerId from '../components/AbbreviatedPeerId';
+import Section from '../future-hopr-lib-components/Section';
+import TextField from '@mui/material/TextField';
+
+function PingPage() {
+  const dispatch = useAppDispatch();
+  const [peerId, set_peerId] = useState('');
+  const [pinging, set_pinging] = useState(false);
+  const [pinged, set_pinged] = useState(false);
+  const [invalidPeerId, set_invalidPeerId] = useState(false);
+
+  const pings = useAppSelector((selector) => selector.node.pings);
+  const aliases = useAppSelector((selector) => selector.node.aliases);
+  const loginData = useAppSelector((selector) => selector.auth.loginData);
+  const {
+    apiEndpoint,
+    apiToken,
+  } = loginData;
+
+  // Get aliases on page load
+  useEffect(() => {
+    if (apiEndpoint && apiToken) {
+      dispatch(
+        nodeActionsAsync.getAliasesThunk({
+          apiEndpoint,
+          apiToken,
+        }),
+      );
+    }
+  }, []);
+
+  const isAlias = (alias: string) => {
+    if (aliases) {
+      return !!aliases[alias];
+    } else return false;
+  };
+
+  const validatePeerId = (peerId: string) => {
+    if (isAlias(peerId)) {
+      if (aliases) {
+        return aliases[peerId];
+      } else return peerId;
+    } else return peerId;
```

Comment:
```suggestion
    if (aliases && isAlias(peerId)) {
        return aliases[peerId];
    } 
    
    return peerId;
```

Reply:
I would do something like this, let me know if there are any issues with this logic

Reply:
Sure! Will update on both sides we use this, it might be good to make this a util function in the future